### PR TITLE
Add pt-br/docs/reference/glossary/aggregation-layer.md

### PR DESCRIPTION
**Pull request by MrErlison**
Signed-off-by: Mr. Erlison <sisal.cordame_0a@icloud.com>

Added translation for portuguese of the glossary term aggregation layer.

### CODE REVIEW

**Comment by edsoncelio** [comment]
Thanks @MrErlison!
/lgtm

@devlware could take a look on this?

**Comment by edsoncelio** [comment]
(adding my LGTM again, removed by mistake - wrong PR)
/lgtm

File: content/pt-br/docs/reference/glossary/aggregation-layer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+date: 2018-10-08
+full_link: /pt-br/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
+short_description: >
+  A camada de agregação permite que você instale APIs adicionais no estilo Kubernetes em seu cluster.
+
+aka: 
+tags:
+- arquitetura
+- extensão
+- operação
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/aggregation-layer.md:13]: This change breaks the glossary, as the canonical tags are currently hardcoded. Could you please revert it?

File: content/pt-br/docs/reference/glossary/aggregation-layer.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+ A camada de agregação permite que você instale APIs adicionais no estilo Kubernetes em seu cluster.
+
+<!--more-->
+
+Depois de configurar o {{< glossary_tooltip text="Servidor de API do Kubernetes" term_id="kube-apiserver" >}} para [suportar APIs adicionais](/docs/tasks/extend-kubernetes/configure-aggregation-layer/), você pode adicionar objetos `APIService` para obter a URL da API adicional.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/aggregation-layer.md:19]: Very minor: "servidor da API" instead of "servidor de API".

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @MrErlison, thank you very much for your contribution! 🎉 

There are some changes required. Right now, I don't see this glossary entry appear in the Portuguese preview, as the canonical tags are not correctly mapped.

After these are fixed, it will be a LGTM. 😃

File: content/pt-br/docs/reference/glossary/aggregation-layer.md (hunk 1 above)
Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/aggregation-layer.md:13]: Sorry about that. I did before your explanation and forgot to fix.

**Comment by stormqueen1990** [comment]
/lgtm

@edsoncelio could you take another look?

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**